src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2265,6 +2265,31 @@ sub identify_stars {
     }
 }
 
+use Inline Python => q{
+
+from Chandra.Time import DateTime
+import numpy as np
+from astropy.table import Table
+from mica.stats.acq_stats import get_stats
+
+ACQS = get_stats()
```

Comment:
This just reads... everything from the table.  It would probably be useful to add a routine in mica acq stats to get this by star.

Reply:
Yes, we should be able to add an index by AGASC ID and make very efficient queries.

Having said that, it isn't completely obvious that total run time will be reduced if you think about making lots of disk queries for individual stars vs one bulk disk query and then memory queries.